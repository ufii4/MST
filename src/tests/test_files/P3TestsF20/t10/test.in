n 5
d 4;3
edge_count
clear
i 2;3;5
i 1;3;6
i 1;4;2
mst
i 0;1;3
i 0;2;3
i 1;2;1.3
i 2;4;2.1
mst
clear
mst
exit
