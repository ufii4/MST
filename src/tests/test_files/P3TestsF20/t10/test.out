success
failure
edge count is 0
success
success
success
success
not connected
success
success
success
success
mst 11.30
success
not connected
